OAuth/FirebaseOAuthUI/FUIOAuth.m

```diff
@@ -117,12 +121,82 @@ - (instancetype)initWithAuthUI:(FUIAuth *)authUI
     _icon = iconImage;
     _scopes = scopes;
     _customParameters = customParameters;
-    _provider = [FIROAuthProvider providerWithProviderID:self.providerID];
     _loginHintKey = loginHintKey;
+    if (![_providerID isEqualToString:@"facebook.com"] && ![_providerID isEqualToString:@"apple.com"]) {
+      _provider = [FIROAuthProvider providerWithProviderID:self.providerID];
+    }
   }
   return self;
 }
 
++ (FUIOAuth *)twitterAuthProvider {
+  return [[FUIOAuth alloc] initWithAuthUI:[FUIAuth defaultAuthUI]
+                               providerID:@"twitter.com"
+                          buttonLabelText:@"Sign in with Twitter"
```

Comment:
do these button label texts need to be localized?

Reply:
Seems we don't localize any strings in FirebaseUI currently. While I think it's a good idea.